.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "voorjaar.windicss-intellisense",
```

Comment:
what is this used for?

Reply:
I used [windicss](https://windicss.org/) (similar to tailwind) in the UI. This plugin adds intellisense and suggests class names. In addition it shows the css a specific class will generate.

Reply:
@anbraten should we mention that at `UI development`?

Reply:
added a new section to `ui development` named `Tools and frameworks`